src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage.Accessor.cs

```diff
@@ -202,7 +202,7 @@ private Task AddWriteTaskAsync(TKey key, Action<SqlConnection> action, Cancellat
             private bool ChecksumsMatch_MustRunInTransaction(SqlConnection connection, long rowId, Checksum checksum, CancellationToken cancellationToken)
             {
                 using var checksumStream = connection.ReadBlob_MustRunInTransaction(DataTableName, ChecksumColumnName, rowId);
-                using var reader = ObjectReader.TryGetReader(checksumStream, cancellationToken);
+                using var reader = ObjectReader.TryGetReader(checksumStream, leaveOpen: false, cancellationToken);
```

Comment:
💡 No need to leave this open since the containing using statement will already dispose of `checksumStream`